bh_modules/s840d_hmi.py

```diff
@@ -0,0 +1,22 @@
+"""
+BracketHighlighter.
+
+Copyright (c) 2013 - 2016 Deathaxe <deathaxe82@gmail.com>
+License: MIT
+"""
+
+
+def compare(name, first, second, bfr):
+    """Ensure correct open is paired with correct close."""
+
+    o = bfr[first.begin:first.end].lower()
+    c = bfr[second.begin:second.end].lower()
+
+    match = False
+    # classes
+    if o in ["//a", "//b", "//g", "//m", "//s"] and c == "//end":
```

Comment:
Instead of creating a new list instance every time the compare method is called, how about creating a static tuple up front?

```
S840D_HMI_CLASSES = ("//a", "//b", "//g", "//m", "//s")

...

if o in S840D_HMI_CLASSES and c == "//end":
```

See the following for reference: https://github.com/facelessuser/BracketHighlighter/blob/master/bh_modules/rubykeywords.py.

We don't really need a dynamic list and tuples are faster to assign to. And this way we only bother allocating the object when the file is loaded, after that all calls to the compare method just references the object.

Everything else looks fine.